src/shared/user-menu.jsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react'
```

Comment:
Please revert the changes to this file, as discussed.

Reply:
should I revert all the changes of user-menu.jsx

Reply:
Yes. I don't think the changes have any functional impact, certainly they don't have anything to do with the issue at hand? Unless there was some reason you proposed them that I don't understand?

Reply:
ok then I open another issue regarding this. when we are clicking the loginof header message, notebook header doesn't showing the avatar of user.

Reply:
I'm not sure if I completely understand but definitely file an issue if you noticed another problem, perhaps with a screenshot demonstrating the issue.